### PR TITLE
Clean Undetermined reads IRMA

### PR DESCRIPTION
**Pull request by senthil10**
An option to clean the undetermined reads and when it is called it will only clean the undetermined reads.

### CODE REVIEW

File: taca/cleanup/cleanup.py [hunk 1]
```diff
@@ -234,14 +236,41 @@ def cleanup_irma(days_fastq, days_analysis, only_fastq, only_analysis, status_db
...
         logger.info("Option 'only_analysis' is given, so will not look for fastq data")
-     
-    if only_analysis:
+    
+    if clean_undetermined:
+        all_undet_files = []
+        for flowcell_dir in flowcell_dir_root:
+            for fc in [d for d in os.listdir(flowcell_dir) if re.match(filesystem.RUN_RE,d)]:
+                fc_abs_path = os.path.join(flowcell_dir, fc)
+                with filesystem.chdir(fc_abs_path):
```
Comment (vezzi) [inline taca/cleanup/cleanup.py]: just checking... if you do `chdir` inside a `with` statement, when you leave the `with` you go back to the original folder? If not then you will start the execution in one folder and then jump around in the filesystem....

Comment (senthil10) [inline taca/cleanup/cleanup.py]: Yes @vezzi , the script goes back to original directory when it leaves the `with` statement here. So in this context, it goes inside each flowcell and then collect the undetermined files (with absolute path) 👍

Comment (vezzi) [inline taca/cleanup/cleanup.py]: great that was my guess

**Comment by vezzi** [comment]
Great job

**Pull request merged**